e3sm_diags/plot/enso_diags_plot.py

```diff
@@ -138,7 +167,7 @@ def plot_scatter(
     plt.ylabel("{} anomaly ({})".format(y["var"], y["units"]))
     plt.legend()
 
-    _save_plot(fig, parameter)
+    _save_plot_scatter(fig, parameter)
```

Comment:
the scatter plots has very different layout as 3 panel maps, use custom _save_plot_scatter here instead.